style.css

```diff
@@ -511,7 +511,7 @@ header {
     margin-top: -20px;
     background-image: none;
     margin-left: 160px;
-    max-width: 800px;
+    max-width: 55%;
```

Comment:
On line 514 we changed max-width to 55%, so about-me paragraph is easier to read.